oraclelinux8/scripts/build-rhsm.sh

```diff
@@ -43,3 +43,47 @@ if [ ! -d /output/oraclelinux8 ]; then
   mkdir /output/oraclelinux8
 fi
 cp -rf /root/rpmbuild/RPMS/* /output/oraclelinux8/
+
+# Clone the subscription-manager-rhsm-certificates repo
+cd /root || exit
+git clone https://github.com/candlepin/subscription-manager-rhsm-certificates.git
+cd /root/subscription-manager-rhsm-certificates || exit
+
+# Add tag with proper version and release
+git tag subscription-manager-rhsm-certificates-$RHSM_VERSION-$RHSM_RELEASE
```

Comment:
Shouldn't this already be tagged upstream? I check to see what the version is by checking the UBI images for each version, so this tag should already exist. We don't want to use latest, we need to use the version used by upstream so it migrates properly.